src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.ts

```diff
@@ -121,12 +130,21 @@ export class RegulatoryOrganisationTypeComponent extends RegisterComponent imple
 
   public onContinueClicked(): void {
     if (this.validateForm()) {
-      this.registrationData.regulatoryOrganisationTypes = this.regulators.value as RegulatoryOrganisationType[];
+      // Set corresponding registration data
+      this.registrationData.regulators = this.regulators.value as Regulator[];
       // eslint-disable-next-line @typescript-eslint/no-empty-function
-      this.router.navigate(['/register-org-new/organisation-services-access']).then(() => {});
+      this.router.navigate(['/register-org-new/organisation-services-access']);
```

Comment:
@johnbenjamin-hmcts - `this.router.navigate()` returns a `Promise`, hence originally I added an empty function to resolve it. Are you sure this is no longer required? If it isn't, please delete the comment on Line 135